json-schema/sigma-filters-schema.json

```diff
@@ -96,13 +70,32 @@
           }
         },
         "selection": {
-          "type": "array",
-          "description": "the filter detection logic",
-          "minItems": 1,
-          "uniqueItems": true,
-          "items": {
-            "type": "string"
-          }
+          "description": "A Search Identifier: A definition that can consist of two different data structures - lists and maps.",
+          "anyOf": [
```

Comment:
And, the `selection` is like `detection` in Sigma rules, hence it should have either of the `map` _or_ the `array` types and shouldn't necessarily be an `array` only.